nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -114,8 +117,47 @@ public static IEnumerable<string> GetProjectPathsFromProject(string projFilePath
                     continue;
                 }
 
+                var fullPath = PathHelper.GetFullPathFromRelative(folderPath, projectPath);
+                if (fullPath.Contains('*'))
```

Comment:
You may want to see if we can just plug in the Microsoft.Extensions.FileGlobbing library to get back the matching files. See https://learn.microsoft.com/en-us/dotnet/core/extensions/file-globbing#get-all-matching-files

Reply:
Thank you! I was not aware of this. This allowed. me to clean this up these changes quite a bit.